fastlane/lib/fastlane/runner.rb

```diff
@@ -115,7 +115,7 @@ def trigger_action_by_name(method_sym, custom_dir, *arguments)
           return self.try_switch_to_lane(method_sym, arguments)
         rescue LaneNotAvailableError
           # No lane, no action, let's at least show the correct error message
-          if PluginManager.new.plugin_is_added_as_dependency?(PluginManager.plugin_prefix + method_sym.to_s)
+          if Fastlane.plugin_manager.plugin_is_added_as_dependency?(PluginManager.plugin_prefix + method_sym.to_s)
```

Comment:
No need to refer back out to the global variable here, right? We're already inside the only expected instance of the class?